src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -309,7 +313,7 @@ export class EmbeddablePanel extends React.Component<Props, State> {
         {!this.props.hideHeader && (
           <PanelHeader
             getActionContextMenuPanel={this.getActionContextMenuPanel}
-            hidePanelTitle={this.state.hidePanelTitle}
+            hidePanelTitle={this.state.hidePanelTitle || !!selfStyledOptions?.hideTitle}
```

Comment:
I am curious if this forcing of hideTitles for this type of viz is required and is a better UX that makes this addition worth it. Especially since we don't do this deviation for other panels, and users can hide titles on a dashboard themselves.  

Regarding the implementation, this probably should have been a part of output$ (not sure) so that embeddable could trigger updates. Also, I noticed that some logic makes the title dependent on hideTiles:

https://github.com/elastic/kibana/blob/a1e33569f2f91c7c7843752da018c1765edc6da9/src/plugins/embeddable/public/lib/embeddables/embeddable.tsx#L21-L23

And because we added `selfStyledOptions?.hideTitle` here, we broke that connection. Now the title works differently with this option for tests and a11y attributes. Not sure if this is very important, but can't be great that we have this different code branch now.

If this config option is needed and we don't have much time, I think it is fine to leave it like this. But if it is possible to revert this, leave this new metric initially without this feature, and take a look at this separately - I think that would be better.

Reply:
@ghudgins @markov00 I would like to follow @Dosant s suggestion here and solve the panel hiding logic in 8.5. What do you think?

Reply:
I believe the design / product intent is this information is extremely redundant so much so that it makes very little sense to have the information in the panel title which is why this was implemented this way. 

we also only get one to define the options of something and that's when the feature is new...if we make it configurable then we will have a very hard time removing it later

while I could be convinced with the approach because it leads to better engineering it comes with the potential of long running baggage of users including this redundant information.

Reply:
Good point @ghudgins - much easier to fix some potential bugs in the way panels are handled than taking away a feature. I changed my mind, +1 for leaving it in the way it is at the moment.

Reply:
agree with @ghudgins, less is more in this case.
I see the @Dosant point about a11y labels, I think this goes into the big bucket of "adding meta-information to the visualization" and we should probably define better how to fix that in the future, but in the current state, as as tech preview, the metric are enough self-explanatory through their internal a11y attributes and the requirement of a global title for the whole block is not required right now IMHO.

I can't speak from the technical/implementation point of view.

one thing I like to underline to @andrewctate: could you please check how an added description works in this case? are we showing the question mark icon on top of the charts? is it reachable? Not a big blocker for that, but I just want to be sure we also considered that